examples/ds-hostnet-split-httploadbalancer/02-rbac.yaml

```diff
@@ -58,6 +58,15 @@ rules:
   - put
   - post
   - patch
+- apiGroups: ["projectcontour.io"]
```

Comment:
Is this file a symlink to ..common/rbac.yaml and github isn't showing me that? If not, could it be made so?

Reply:
It is not, and definitely could be.